src/tally.F90

```diff
@@ -624,38 +597,67 @@ subroutine score_general(p, t, start_index, filter_index, i_nuclide, &
 
 
       case (SCORE_KAPPA_FISSION)
+        ! Determine kappa-fission cross section on the fly. The ENDF standard
+        ! (ENDF-102) states that MT 18 stores the fission energy as the Q_value
+        ! (fission(1))
+
+        score = ZERO
+
         if (t % estimator == ESTIMATOR_ANALOG) then
           if (survival_biasing) then
             ! No fission events occur if survival biasing is on -- need to
             ! calculate fraction of absorptions that would have resulted in
             ! fission scale by kappa-fission
-            if (micro_xs(p % event_nuclide) % absorption > ZERO) then
-              score = p % absorb_wgt * &
-                   micro_xs(p % event_nuclide) % kappa_fission / &
-                   micro_xs(p % event_nuclide) % absorption
-            else
-              score = ZERO
-            end if
+            associate (nuc => nuclides(p % event_nuclide))
+              if (micro_xs(p % event_nuclide) % absorption > ZERO .and. &
+                   nuc % fissionable) then
+                score = p % absorb_wgt * &
+                     nuc%reactions(nuc%index_fission(1))%Q_value * &
```

Comment:
This is a good example of my laziness! I'll at least make it consistent on a single statement.